README.md

```diff
@@ -46,21 +45,9 @@ Skycoin is small part of OP Redecentralize and OP Darknet Plan.
 
 ## Installation
 
-### Installing Go
+### Installation and Setup
 
```

Comment:
Keep comment about the required go version (1.9).  Go developers do not need to read installation instructions and only care which version is required.

Reply:
Should specify `go` version at link or `Installation and Setup` subtitle?

Reply:
Both